seam/seam.py

```diff
@@ -48,11 +89,11 @@ def __init__(
             workspace_id=workspace_id,
             endpoint=endpoint,
         )
-        defaults = {"wait_for_action_attempt": wait_for_action_attempt}
+        self.defaults = {"wait_for_action_attempt": wait_for_action_attempt}
```

Comment:
Why is this added to self now? Do we need it, and if so, maybe it should be private?

Reply:
Added this because I double checked the [JS SDK](https://github.com/seamapi/javascript-http/blob/295dc1707b8e6e1fdf82d53311e0f152ac74a11b/src/lib/seam/connect/seam-http.ts#L53) and it assigns `defaults` as an instance variable. Hence the change.